library/package.json

```
@@ -39,7 +39,10 @@
     "./README.md",
     "./LICENSE"
   ],
-  "sideEffects": false,
+  "sideEffects": [
+    "./style/default.css",
+    "./style/default.min.css"
```

```suggestion
    "./styles/default.css",
    "./styles/default.min.css"
```

`styles`, not `style` :) I will push it.